android-lib/src/androidTest/java/com/kinvey/androidTest/store/user/MICLoginTest.java

```diff
@@ -195,13 +195,13 @@ public void onReadyToRender(String myURLToRender) {
 
     @Test
     public void testMICDefaultAPIVersion() throws IOException {
-        assertEquals("v3", client.getMICApiVersion());
+        assertEquals("v3", client.getMicApiVersion());
```

Comment:
again try use `@JvmName` here

Reply:
Used `@JvmName` converted to old method name for `Java` classes.